packages/engine/CONTRIBUTING.md

```diff
@@ -110,6 +110,14 @@ So you've found a bug, next up is collecting useful information to give us to in
 
 > TODO
 
+We are using `cargo fmt` and `cargo clippy` to apply linting rules to our Rust codebase. In order to run those, you need to install them first (unless they are already installed, which is the default):
```

Comment:
Not necessary at all, but might be nice to link to [rustfmt](https://github.com/rust-lang/rustfmt) and [clippy](https://github.com/rust-lang/rust-clippy)